core/templates/components/profile-link-directives/profile-link-image.component.html

```diff
@@ -0,0 +1,5 @@
+<a *ngIf="isUsernameLinkable()" [href]="/profile/username">
```

Comment:
The`[ href]` here is wrong. It should be something like:
```
[href]=" '/profile/' + username"
```

Remember that when we use `[]` on an attribute, it will look for the variable inside the `""` you mention. So that is why I added single quotes.